### PR TITLE
vcl_reload as celery task

### PR DESCRIPTION
**Pull request by jtaczanowski**

### CODE REVIEW

**Comment by szymi-** [comment]
+1

File: vaas-app/src/vaas/cluster/cluster.py [hunk 1]
```diff
@@ -36,18 +46,26 @@ def __init__(self, timeout=1, max_workers=30):
...
-            raise
+            for cluster in clusters:
+                cluster.reload_timestamp = start_processing_time
+                cluster.save()
+        except VclLoadException as e:
+            error_timestamp = timezone.now()
+            self.logger.error('Loading error: {} - rendered vcl-s not used'.format(e))
+            for cluster in clusters:
+                cluster.error_timestamp = error_timestamp
+                cluster.last_error_info = str(e)
```
Comment (ziollek) [inline vaas-app/src/vaas/cluster/cluster.py]: should be limit to max field size

File: vaas-app/src/vaas/cluster/cluster.py [hunk 2]
```diff
@@ -36,18 +46,26 @@ def __init__(self, timeout=1, max_workers=30):
...
-            self.logger.error("Loading error - rendered vcl-s not used")
-            raise
+            for cluster in clusters:
+                cluster.reload_timestamp = start_processing_time
+                cluster.save()
+        except VclLoadException as e:
+            error_timestamp = timezone.now()
+            self.logger.error('Loading error: {} - rendered vcl-s not used'.format(e))
+            for cluster in clusters:
+                cluster.error_timestamp = error_timestamp
```
Comment (ziollek) [inline vaas-app/src/vaas/cluster/cluster.py]: should use start_processing_time

**Comment by ziollek** [comment]
+1

**Pull request merged**